Testing/xASL_test_Summarize.m

```diff
@@ -0,0 +1,110 @@
+function [summary, summaryTable] = xASL_test_Summarize(ResultsDir)
```

Comment:
A new function or copied from elsewhere?

Reply:
This is a new Function.